packages/components/select-v2/src/defaults.ts

```diff
@@ -256,8 +264,35 @@ export const OptionProps = buildProps({
     type: definePropType<Option>(Object),
     required: true,
   },
-  index: Number,
+  index: {
+    type: Number,
+  },
   style: Object,
   selected: Boolean,
   created: Boolean,
 } as const)
+
```

Comment:
Multiple repeated rule comment markers can be written in a different way.
```ts
/* eslint-disable @typescript-eslint/no-unused-vars */
......
/* eslint-enable @typescript-eslint/no-unused-vars */
```

Reply:
That's neat👍